src/components/application_manager/include/application_manager/application.h

```diff
@@ -657,7 +657,8 @@ class Application : public virtual InitialApplicationData,
    * @brief Wakes up streaming process for application
    * @param service_type Type of streaming service
```

Comment:
Probably add a `@param` line in the function brief explaining what `timer_len` is

Reply:
Added in https://github.com/smartdevicelink/sdl_core/pull/3057/commits/c61c05654ee2d4919b03e0e539eb86ba030f3aba